test/functorch/test_control_flow.py

```diff
@@ -460,16 +465,20 @@ def f(x):
             return cond(pred, true_fn, false_fn, [x])
 
         example_input = torch.ones(4, 5)
-        try:
-            example_input_func = to_fun(example_input)
-            torch._enable_functionalization(reapply_views=False)
-            with self.assertRaisesRegex(UnsupportedAliasMutationException, "One of torch.cond branch"):
+        example_input_func = to_fun(example_input)
+        with self.assertRaisesRegex(UnsupportedAliasMutationException, "One of torch.cond branch"):
+            try:
+                torch._enable_functionalization(reapply_views=False)
```

Comment:
Why did this change?

Reply:
I remeber it is due to PYTORCH_TEST_WITH_DYNAMO=1 flag, let me check if i can remove the change.